src/Features/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.Features.csproj

```
@@ -67,7 +67,7 @@
     <PublicAPI Include="PublicAPI.Unshipped.txt" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Humanizer.Core" Version="$(HumanizerCoreVersion)" PrivateAssets="all" />
+    <PackageReference Include="Humanizer.Core" Version="$(HumanizerCoreVersion)" />
```

❗️  *At minimum*, this needs to use `PrivateAssets="compile"` to avoid making Humanizer part of the public API of this package.

```suggestion
    <PackageReference Include="Humanizer.Core" Version="$(HumanizerCoreVersion)" PrivateAssets="compile" />
```

---

I agree, makes sense - changed it accordingly